sodiumpp/include/sodiumpp/locked_string.h

```diff
@@ -22,13 +22,13 @@ class locked_string final
              * for move construction for std::string use static methods:
              * move_from_locked_string() and move_from_not_locked_string
              */
-            explicit locked_string(std::string && str) noexcept; ///< only move, not call mlock()
```

Comment:
Good idea, we avoid by-accident use of locked string from non-locked memory which probably would be not what caller would want.